src/jobflow_remote/cli/flow.py

```diff
@@ -127,11 +131,30 @@ def delete(
             help="Also delete the outputs of the Jobs in the output Store",
         ),
     ] = False,
+    delete_files: Annotated[
+        bool,
+        typer.Option(
+            "--files",
+            "-fi",
+            help="Also delete the files on the worker",
+        ),
+    ] = False,
+    delete_all: Annotated[
+        bool,
+        typer.Option(
+            "--all",
+            "-a",
+            help="enable --output and --files",
+        ),
+    ] = False,
 ) -> None:
     """Permanently delete Flows from the database"""
     check_incompatible_opt({"start_date": start_date, "days": days, "hours": hours})
     check_incompatible_opt({"end_date": end_date, "days": days, "hours": hours})
 
+    if delete_all:
+        delete_files = delete_output = True
```

Comment:
One thing I'm wondering is: if we delete the outputs, but not the files, we somehow have no way to access (easily) these files ... Should we either
- always delete the files when we delete the outputs (maybe with an option to keep the files?), but maybe it's dangerous ..
- delete most of the content of the dict in the database, but keep uuid, index, db_id and run directory somehow ?
- do nothing

In any case, I think we should document this carefully, whichever option we choose above.

Reply:
Good point. I think the preferred way would depend a lot on users preferences, so probably we should have the least invasive one as default.

Reply:
So maybe something like the second option ? i.e. "delete most of the content of the dict in the database, but keep uuid, index, db_id and run directory somehow ?"

Reply:
Looking more into this, the problem with that approach is that the changes to the output documents are done through the JobStore (and cannot be otherwise). Since the maggma Store only allows full deletion or updates, it means that to leave only a minimal version of the outputs the documents should all be fetched from the Store, cleaned up and resinserted into the Store. If there are many this may be an expensive operation and I am not sure if it is worth.

Reply:
I propose to keep this as it is for now and have an issue to keep track of it and see if we should do something.